rockcraft/utils.py

```diff
@@ -144,12 +144,12 @@ def parse_command(command: str) -> tuple[list[str], list[str]]:
             if arg == "]":
                 in_brackets = False
                 continue
-            args.append(arg)
+            args.append(arg)  # type: ignore[union-attr]
```

Comment:
Instead of disabling the linting I think we should instead add an assertion `assert args is not None` or similar
@lengau this is in line with what we discussed during the sprint right?